public/locales/pt/editor.json

```diff
@@ -0,0 +1,16 @@
+{
+	"editor-desktop-optimized": "Esse site é otimizado para o desktop",
+	"editor-visit-desktop": "Por favor visite readme.so em um desktop para criar o seu readme!",
+	"nav-download": "Download",
+	"download-readme-generated": "Readme Gerado!",
+	"download-reach-out": "Obrigado por usar o readme.so! Sinta-se livre para entrar em contato comigo no",
+	"download-feedback": "com qualquer feedback.",
+	"download-coffee": "Se você achou esse projeto útil, considere me dar um suporte com uma xícara de café!",
```

Comment:
```suggestion
	"download-coffee": "Se você achou este projeto útil, considere apoiar-me com uma xícara de café!",
```

Reply:
I think this one is ok too
The `apoiar-me` sound too formal IMO

Reply:
I understand. However, "dar um suporte" sounds odd - would "dar suporte" work for you?

Reply:
I would go with `me apoiar`, WDYT?

Reply:
Sounds good to me!